nixos/modules/services/search/curator.nix

```diff
@@ -0,0 +1,93 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+    cfg = config.services.curator;
+    curatorConfig = pkgs.writeTextFile {
+      name = "config.yaml";
+      text = ''
+        ---
+        # Remember, leave a key empty if there is no value.  None will be a string,
+        # not a Python "NoneType"
+        client:
+          hosts: ${builtins.toJSON cfg.hosts}
+          port: 9200
```

Comment:
Shouldn't the port also be configurable? We should probably default it to [services.elasticsearch.port](https://github.com/NixOS/nixpkgs/blob/72f3a5cf5ce77ac8c4d431f104e06fe94cdf7366/nixos/modules/services/search/elasticsearch.nix#L72).